mlir/test/Integration/Dialect/Linalg/CPU/ArmSME/fill-2d.mlir

```diff
@@ -4,9 +4,9 @@
 // RUN:   -lower-vector-mask \
 // RUN:   -one-shot-bufferize="bufferize-function-boundaries" \
 // RUN:   -enable-arm-streaming="streaming-mode=streaming-locally za-mode=new-za" \
-// RUN:   -convert-vector-to-arm-sme -convert-arm-sme-to-scf \
+// RUN:   -convert-vector-to-arm-sme -convert-arm-sme-to-scf -allocate-arm-sme-tiles  \
```

Comment:
you mentioned `-allocate-arm-sme-tiles` can happen immediately after `-convert-vector-to-arm-sme`, is this the case here?